lib/utils.js

```diff
@@ -36,7 +36,9 @@ export function grammarToLanguage(grammar: ?atom$Grammar) {
     l => l.toLowerCase() === grammarLanguage
   );
 
-  return kernelLanguage ? kernelLanguage.toLowerCase() : grammarLanguage;
```

Comment:
Oh! I see my confusion. Since we are mapping by kernel names, we need to return the kernel name!